e2e_tests/e2e_test.py

```diff
@@ -9,9 +9,22 @@
 from selenium.webdriver.common.keys import Keys
 import time
 from dotenv import load_dotenv
+from todo_app.user.user import User
+from flask_login import utils
+
+
+@pytest.fixture(scope='module')
+def monkeymodule():
+    from _pytest.monkeypatch import MonkeyPatch
+    mpatch = MonkeyPatch()
+    yield mpatch
+    mpatch.undo()
 
 @pytest.fixture(scope='module')
-def test_app():
+def test_app(monkeymodule):
+    def get_user():
+        return User("19879648")

```

Comment:
It would be nice to extract out the writer into an environment variable so that the tests/code don't both need changing if the default writer does